test_runner/regress/test_combocid.py

```diff
@@ -0,0 +1,37 @@
+from fixtures.neon_fixtures import NeonEnvBuilder
+
+
+def test_combocid(neon_env_builder: NeonEnvBuilder):
+    env = neon_env_builder.init_start()
+    endpoint = env.endpoints.create_start("main")
+
+    conn = endpoint.connect()
+    cur = conn.cursor()
+    n_records = 100000
+
+    cur.execute("create table t(id integer, val integer)")
```

Comment:
Also, there was a TODO list proposed

> This looks correct. TODOs:
 > - Add tests to cover each of the cases, INSERT, UPDATE, DELETE, LOCK, MULTI_INSERT
 > - There's a funny case within heap_update, where it writes a LOCK record for the tuple first, if it has a toast tuple. Not sure of the details, but it looks correct. Let's just make sure we have a test to cover that too.

I'm not sure which part of the test covers what? And does it do it at all? Some comments could be helpful

Reply:
Currently, the ` test_combocid_delete` test covers the DELETE case. The other cases are not covered.

Reply:
I have added tests for other operations